### PR TITLE
Hcap v2.19 hotfixes

### PR DESCRIPTION
**Pull request by dbayly-freshworks**
Fixed validation filters and typos

### CODE REVIEW

File: client/src/constants/validation.js [hunk 1]
```diff
@@ -376,13 +376,16 @@ export const ArchiveHiredParticipantSchema = yup.object().shape({
...
   }),
   status: yup.string().when('type', {
     is: 'employmentEnded',
     then: yup.string().required('Please include a status').oneOf(archiveStatusOptions),
   }),
-  endDate: yup.date().required('Please enter the date this participant was removed.'),
+  endDate: yup
+    .date()
+    .required('Please enter the date this participant was removed.')
+    .test('is-present', 'Not a valid date in the past', validatePastDateString),
```
Comment (Zenen-freshworks) [inline client/src/constants/validation.js:388]:
```suggestion
    .test('is-present', 'Invalid entry. Date must be in the past.', validatePastDateString),
```

File: client/src/constants/validation.js [hunk 2]
```diff
@@ -275,7 +275,7 @@ export const InterviewingFormSchema = yup
     contactedDate: yup
       .string()
       .required('Date of contact is required')
-      .test('is-date', 'Not a valid date in the past', validatePastDateString),
+      .test('is-date', 'Not a valid date. Date is in the past', validatePastDateString),
```
Comment (Zenen-freshworks) [inline client/src/constants/validation.js:278]:
```suggestion
      .test('is-date', 'Invalid entry. Date must be in the past.', validatePastDateString),
```

File: server/validation.js [hunk 3]
```diff
@@ -583,6 +583,7 @@ const ParticipantStatusChange = yup
           endDate: yup
             .string()
             .test('is-date', 'Not a valid date', validateDateString)
+            .test('is-present', 'Not a valid date in the past', validatePastDateString)
```
Comment (Zenen-freshworks) [inline server/validation.js:586]:
```suggestion
            .test('is-present', 'Invalid entry. Date must be in the past.', validatePastDateString)
```

**Review by Zenen-freshworks — COMMENTED** [review]
Just a couple of fixes to clarify validation errors

**Review by Zenen-freshworks — APPROVED** [review]

**Review by fwpushan — APPROVED** [review]
LGTM

**Pull request merged**